vic-class08-assignment/server.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const http = require('http');
+const fs = require('fs');
+const stream = require('stream');
+
+http.createServer((req, res) => {
+  if(req.url === '/notes' && req.method === 'GET') {
+    fs.readdirSync(__dirname + '/data', (err, data) => {
+      if (err) {return res.end('err');}
+    });
+    res.writeHead(200, {'Content-Type': 'text/plain'});
+    let file = fs.readdirSync(__dirname + '/data');
+    console.log(file);
+    var stringifiedArray = file.toString();
+    console.log('working');
+    res.write(stringifiedArray + '\n');
+    return res.end();
+  }
+  if(req.url === '/notes' && req.method === 'POST') {
+    fs.readdir(__dirname + '/data', (err, data) => {
+      if (err) {return res.end('err');}
+    });
+    let fileNum = fs.readdirSync(__dirname + '/data').length + 1;
```

Comment:
Same comment as above.  Also make sure you know the difference between readdir and readdirSync.